src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/include/index.js

```diff
@@ -0,0 +1,23 @@
+const convertInclude = async block => {
+  const { href, url, ...rest } = block;
+
+  const blockMappings = {
+    idt2: 'idt2',
+    include: 'include',
+  };
+  const type = href.split('/')[1];
+
+  if (!blockMappings[type]) {
+    return null;
+  }
+  return {
+    type,
```

Comment:
Wouldn't we want to return the block that `type` maps to, rather than `type` itself? I know the values are identical but otherwise what's the point of `blockMappings`?

Reply:
we're using `blockMappings` to isolate `IIDT1` types from being preprocessed